lib/membrane/video_compositor/scene.ex

```diff
@@ -2,6 +2,13 @@ defmodule Membrane.VideoCompositor.Scene do
   @moduledoc """
   Structure representing a top level specification of what is Video Compositor
   supposed to render.
+
+  The main part of the Scene are `Membrane.VideoCompositor.Scene.Object`s
+  and interactions between them. There are two kinds of Objects:
+  - `Membrane.VideoCompositor.Scene.Object.Texture` - single input object,
+  taking frames and applying a series of transformations onto it.
+  - `Membrane.VideoCompositor.Scene.Object.Layout` - combining
+  frames from multiple inputs into a single output.
```

Comment:
From this description, it's hard to identify that `Texture` is a "wrapper" struct and `Layout` is a specification for custom object implementation.
```elixir
objects = [
  texture_object_name: %Texture{
    input: :some_input_name,
    transformations: [@cropping, @corners_rounding]
  },
  layout_object_name: %SomeLayoutImplementation{
    @some_layout_impl_spec
    | inputs: %{
        input_name_1: :some_input_name,
        ...
        input_name_n: :some_input_name,
      },
      resolution: %Resolution{width: 1920, height: 1080}
  }
]
```

Reply:
I don't think it is necessary here, as moduledocs of Texture and Layouts are one click away. Plus, in that case it would be nice to explain differences between layouts and transformations.

Reply:
Ok